src/view/foreach.rs

```diff
@@ -353,6 +354,50 @@ where
 
         renderables
     }
+
+    fn handle_event(
```

Comment:
Oops...I guess this is what I get for being lazy and adding a default impl for handle_event 😅